CHANGELOG.md

```diff
@@ -1,3 +1,94 @@
+# v0.8.4
+
+## Announcing Akri v0.8.4!
+Akri v0.8.4 is a pre-release of Akri.
+
+To find out more about Akri, check out our [documentation](https://docs.akri.sh/) and start
+[contributing](https://docs.akri.sh/community/contributing) today!
+
+## New Features
+The v0.8.4 release contains the following major changes: 
+
+1. **Support for Kubernetes Job brokers** (https://github.com/project-akri/akri/pull/437). Now Akri has support for deploying Jobs to devices discovered by the Akri Agent. Previously, Akri only supported deploying Pods that were not intended to terminate (and would be restarted if they did). Adding Jobs enables more device use scenarios. More background can be found in the [Jobs proposal](https://github.com/project-akri/akri-docs/blob/main/proposals/job-brokers.md). This is a **breaking change** as it required changes to Akri's Configuration CRD.
+2. Fix to re-enable **applying multiple Configurations that use the same Discovery Handler** (https://github.com/project-akri/akri/pull/432). This adds back functionality that was removed in `v0.6.5` when enabling Akri's new extensibility model. 
+3. Akri depends on `crictl` to track whether Pods deployed by the Akri Controller are still running. This release adds new functionality (https://github.com/project-akri/akri/pull/418) such that **crictl is pre-installed in the Agent container**  so that it does not need to be installed on each node.
+
+**Fixes, features, and optimizations**
+- fix: Make debug echo capacity configurable (https://github.com/project-akri/akri/pull/419)
+- fix: Return okay if get 404 when trying to delete an Instance (https://github.com/project-akri/akri/pull/420)
+- opt: Update .NET dependencies, removing vulnerabilities and reducing size (https://github.com/project-akri/akri/pull/422)
+- feat: Execute test workloads based on labels instead of flags in PR titles (https://github.com/project-akri/akri/pull/426)
+- opt: Set K8s distribution with Helm to simplify choosing container runtime socket (https://github.com/project-akri/akri/pull/427)
+- fix: Fix all clippy errors and update dependency versions (https://github.com/project-akri/akri/pull/442)
+
+View the [full change log](https://github.com/project-akri/akri/compare/v0.7.0...0.8.4)
+
+## Breaking Changes
+Akri's Configuration CRD has been updated to support Job brokers. If Akri has previously been installed on a cluster, delete the previous Configuration CRD before installing the latest version of Akri:
+
+```sh
+kubectl delete crd configurations.akri.sh
+```
+
+## Known Issues
+N/A
```

Comment:
should we mention kubernetes 22, 23 here?

Reply:
I dont see that as an issue with Akri just that we need to update the install script in our pipeline